src/editor/DOMEventHandlers.ts

```diff
@@ -121,6 +122,29 @@ export class DOMEventHandlers {
     )
   }
 
+  static setFontColor(payload: string) {
+    DOMEventHandlers.getEditorInstance().command.executeColor(payload)
+  }
+
+  static highlightText(payload: string) {
+    DOMEventHandlers.getEditorInstance().command.executeHighlight(payload)
+  }
+
+  static setFont(payload: string) {
+    DOMEventHandlers.getEditorInstance().command.executeFont(payload)
+  }
+
+  static setSize(payload: number) {
+    DOMEventHandlers.getEditorInstance().command.executeSize(payload)
+  }
+
+  static increaseFontSize() {
```

Comment:
I think we already had `handleFontSize` for increasing/decreasing fonts. Can you check?
If it is already there, then please cleanup the unused methods.